[2025-04-17T11:50:24.570+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-17T11:50:25.500+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: beer_data_processing.check_file_exists scheduled__2025-04-17T05:00:00+00:00 [queued]>
[2025-04-17T11:50:24.931+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: beer_data_processing.check_file_exists scheduled__2025-04-17T05:00:00+00:00 [queued]>
[2025-04-17T11:50:24.934+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-04-17T11:50:25.419+0000] {taskinstance.py:2890} INFO - Executing <Task(FileSensor): check_file_exists> on 2025-04-17 05:00:00+00:00
[2025-04-17T11:50:26.677+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'beer_data_processing', 'check_file_exists', 'scheduled__2025-04-17T05:00:00+00:00', '--job-id', '902', '--raw', '--subdir', 'DAGS_FOLDER/beer_data_processing_dag.py', '--cfg-path', '/tmp/tmp0pzn4b6g']
[2025-04-17T11:50:27.080+0000] {standard_task_runner.py:105} INFO - Job 902: Subtask check_file_exists
[2025-04-17T11:50:37.357+0000] {task_command.py:467} INFO - Running <TaskInstance: beer_data_processing.check_file_exists scheduled__2025-04-17T05:00:00+00:00 [running]> on host a0d0456bae2c
[2025-04-17T11:50:38.263+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=39554) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-17T11:50:38.387+0000] {standard_task_runner.py:72} INFO - Started process 40037 to run task
[2025-04-17T11:50:41.536+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='beer_data_processing' AIRFLOW_CTX_TASK_ID='check_file_exists' AIRFLOW_CTX_EXECUTION_DATE='2025-04-17T05:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-17T05:00:00+00:00'
[2025-04-17T11:50:41.668+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-04-17T11:50:41.679+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-04-17T11:50:41.682+0000] {logging_mixin.py:190} INFO - Current task name:check_file_exists state:running start_date:2025-04-17 11:50:25.519573+00:00
[2025-04-17T11:50:41.683+0000] {logging_mixin.py:190} INFO - Dag name:beer_data_processing and current dag run status:running
[2025-04-17T11:50:41.697+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-17T11:50:41.739+0000] {baseoperator.py:423} WARNING - FileSensor.execute cannot be called outside TaskInstance!
[2025-04-17T11:50:41.904+0000] {base.py:84} INFO - Retrieving connection 'dataset_folder'
[2025-04-17T11:50:41.912+0000] {filesystem.py:109} INFO - Poking for file /opt/***/dags/data/beer_profile_and_ratings.csv
[2025-04-17T11:50:41.939+0000] {filesystem.py:113} INFO - Found File /opt/***/dags/data/beer_profile_and_ratings.csv last modified: 20250416145823
[2025-04-17T11:50:41.942+0000] {base.py:339} INFO - Success criteria met. Exiting.
[2025-04-17T11:50:41.954+0000] {filesystem.py:109} INFO - Poking for file /opt/***/dags/data/beer_profile_and_ratings.csv
[2025-04-17T11:50:41.958+0000] {filesystem.py:113} INFO - Found File /opt/***/dags/data/beer_profile_and_ratings.csv last modified: 20250416145823
[2025-04-17T11:50:42.190+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-17T11:50:42.197+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=beer_data_processing, task_id=check_file_exists, run_id=scheduled__2025-04-17T05:00:00+00:00, execution_date=20250417T050000, start_date=20250417T115025, end_date=20250417T115042
[2025-04-17T11:50:42.487+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-04-17T11:50:42.520+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-04-17T11:50:42.545+0000] {logging_mixin.py:190} INFO - Dag name:beer_data_processing queued_at:2025-04-17 11:36:11.494718+00:00
[2025-04-17T11:50:42.549+0000] {logging_mixin.py:190} INFO - Task hostname:a0d0456bae2c operator:FileSensor
[2025-04-17T11:50:42.816+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-17T12:01:06.253+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
[2025-04-17T12:01:08.463+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-17T12:01:08.571+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
